_posts/2024-10-07-odero-kdr-paper.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+title:  "Odero et al. 2024 - Discovery of Knock-Down Resistance in the Major African Malaria Vector _Anopheles funestus_"
```

Comment:
Not sure if we can use markdown in frontmatter? Maybe I'll merge and lets see...